src/pages/signup/components/SingupAgree.tsx

```
@@ -33,11 +36,21 @@ const SingupAgree = ({ onNext }: SignupProps) => {
       alert('모든 필수 항목에 동의해야 합니다.');
+      return;
+    }
+
+    try {
+      await verifyMutation.mutateAsync({
+        email: data.email,
+        type: 'SIGN_UP',
+      });
+      setEmail(data.email);
+      onNext();
+    } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)